docs/source/release/v3.14.0/framework.rst

```diff
@@ -40,6 +40,7 @@ New Algorithms
 ##############
 
 - :ref:`MatchSpectra <algm-MatchSpectra>` is an algorithm that calculates factors to match all spectra to a reference spectrum.
+- :ref:`MaskBinsIf <algm-MaskBinsIf>` is an algorithm to mask bins according to criteria specified as a muparser expression.
```

Comment:
I think you need to make it clear that the data is also being zero'd out as part of the masking process.